app/controllers/categories_controller.rb

```diff
@@ -7,7 +7,8 @@ class CategoriesController < ApplicationController
   before_action :check_size, only: %i[new create edit update]
 
   def index
-    @pagy, @categories = pagy(Category.all.order(sort_params))
+    list = current_user&.admin_type? ? Category.all : Category.active
```

Comment:
Are we using the same controller for regular user and for admin? It will be better to have CategoriesController for users and Admin::CategoriesController - for admin.

Reply:
Yes, we are. 
Then I think we need to make another pull request for this task

Reply:
I'm going to do it